positron/comms/variables-backend-openrpc.json

```diff
@@ -189,6 +189,57 @@
 				},
 				"required": false
 			}
+		},
+		{
+			"name": "query_variable_data",
+			"summary": "Query variable data",
+			"description": "Request a data summary for a variable or variables.",
+			"params": [
+				{
+					"name": "path",
+					"description": "The path to the variable to inspect, as an array of access keys.",
+					"schema": {
+						"type": "array",
+						"items": {
+							"type": "string"
+						}
+					}
+				},
+				{
+					"name": "query_types",
+					"description": "A list of types to summarize.",
+					"schema": {
+						"type": "array",
+						"items": {
+							"type": "string"
+						}
+					}
+				}
+			],
+			"result": {
+				"schema": {
+					"name": "summarized_data",
+					"description": "Result of the summarize operation",
+					"type": "object",
+					"properties": {
+						"children": {
```

Comment:
This is returning a different return type right now (with the schema and column profiles, so a lot more complex). I think to avoid having to drag along the schema and profile result type (and mainly having to expose these in the Positron runtime / extHost API) we can just return the schema and profiles as a serialized JSON string to sidestep this issue for now -- it would be good to make these results well-typed everywhere but there's a bunch of plumbing needed).